projects/gnomad/src/server/server.js

```diff
@@ -8,6 +8,15 @@ const port = process.env.PORT || 80
 const app = express()
 app.use(compression())
 
+// Redirect HTTP requests to HTTPS
+app.use((request, response, next) => {
+  if (request.get('x-forwarded-proto') === 'http') {
```

Comment:
Looks like there is an express check for security https://expressjs.com/en/api.html#req.secure

Reply:
I'm not sure if that will work, since the connection from load balancer to express is unencrypted.

Reply:
I was also thinking that this way would only redirect in the load balanced environment, while still allowing server.js to be run on HTTP locally.

Reply:
Do we need to support using this script non-https locally if we have webpack for development? Also, looks like it only checks 'https' == req.protocol;`? I can help with testing tomorrow too, should be able to spin up a local nginx pretty quick, or is there another way to test? Should we just do it in prod during non-peak time?